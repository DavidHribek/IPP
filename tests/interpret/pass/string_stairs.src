.ippCODE18

DEFVAR GF@symb
MOVE GF@symb string@_

DEFVAR GF@var
MOVE GF@var string@

DEFVAR GF@counter # counter end
MOVE GF@counter int@0

LABEL start
JUMPIFEQ end GF@counter int@100 # konec
ADD GF@counter GF@counter int@1 # pricteni citace

CONCAT GF@var GF@var GF@symb # pridej schod
WRITE GF@var # vypis radu schodu

JUMP start
LABEL end