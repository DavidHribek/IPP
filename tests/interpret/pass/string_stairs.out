_
__
___
____
_____
______
_______
________
_________
__________
___________
____________
_____________
______________
_______________
________________
_________________
__________________
___________________
____________________
_____________________
______________________
_______________________
________________________
_________________________
__________________________
___________________________
____________________________
_____________________________
______________________________
_______________________________
________________________________
_________________________________
__________________________________
___________________________________
____________________________________
_____________________________________
______________________________________
_______________________________________
________________________________________
_________________________________________
__________________________________________
___________________________________________
____________________________________________
_____________________________________________
______________________________________________
_______________________________________________
________________________________________________
_________________________________________________
__________________________________________________
___________________________________________________
____________________________________________________
_____________________________________________________
______________________________________________________
_______________________________________________________
________________________________________________________
_________________________________________________________
__________________________________________________________
___________________________________________________________
____________________________________________________________
_____________________________________________________________
______________________________________________________________
_______________________________________________________________
________________________________________________________________
_________________________________________________________________
__________________________________________________________________
___________________________________________________________________
____________________________________________________________________
_____________________________________________________________________
______________________________________________________________________
_______________________________________________________________________
________________________________________________________________________
_________________________________________________________________________
__________________________________________________________________________
___________________________________________________________________________
____________________________________________________________________________
_____________________________________________________________________________
______________________________________________________________________________
_______________________________________________________________________________
________________________________________________________________________________
_________________________________________________________________________________
__________________________________________________________________________________
___________________________________________________________________________________
____________________________________________________________________________________
_____________________________________________________________________________________
______________________________________________________________________________________
_______________________________________________________________________________________
________________________________________________________________________________________
_________________________________________________________________________________________
__________________________________________________________________________________________
___________________________________________________________________________________________
____________________________________________________________________________________________
_____________________________________________________________________________________________
______________________________________________________________________________________________
_______________________________________________________________________________________________
________________________________________________________________________________________________
_________________________________________________________________________________________________
__________________________________________________________________________________________________
___________________________________________________________________________________________________
____________________________________________________________________________________________________
